src/map/lua/luautils.cpp

```diff
@@ -1517,27 +1517,61 @@ namespace luautils
         lua_pushnil(LuaHandle);
         lua_setglobal(LuaHandle, "onEventUpdate");
 
+        bool luaFileFound = false;
```

Comment:
I can do that. I should have to begin with, but the previous logic was itself copy/paste and I was getting things working first (bad excuse). I'll clean it up today.